vmm/src/lib.rs

```diff
@@ -613,12 +613,17 @@ impl Vmm {
         if let Some(device_idx) = self.drive_handler_id_map.get(drive_id) {
             match self.epoll_context.get_device_handler(*device_idx) {
                 Ok(handler) => {
-                    handler.handle_event(
+                    let r = handler.handle_event(
                         virtio::block::FS_UPDATE_EVENT,
                         *device_idx as u32,
                         EpollHandlerPayload::DrivePayload(disk_image),
                     );
-                    Ok(())
+                    match r {
```

Comment:
`match` directly on the result of `handle_event` instead of moving it around through a variable